src/main/java/com/yelp/nrtsearch/server/field/AtomFieldDef.java

```diff
@@ -150,4 +150,10 @@ public Query getRangeQuery(RangeQuery rangeQuery) {
           "Only SORTED or SORTED_SET doc values are supported for range queries: " + getName());
     }
   }
+
+  @Override
+  public Query getPrefixQuery(PrefixQuery prefixQuery, MultiTermQuery.RewriteMethod rewriteMethod) {
+    return new org.apache.lucene.search.PrefixQuery(
+        new Term(prefixQuery.getField(), prefixQuery.getPrefix()), rewriteMethod);
+  }
```

Comment:
You can add a call to `self.verifySearchable()` to replace the check in the query node mapper